test/integration/DependenciesFromConfigProviderExtractionTrait.php

```diff
@@ -4,24 +4,23 @@
 
 namespace Boesing\Psr\Http\Message\Multipart;
 
+use Laminas\ServiceManager\ConfigInterface;
 use PHPUnit\Framework\TestCase;
 
 /**
  * @see TestCase
  *
  * @psalm-require-extends TestCase
+ * @psalm-import-type ServiceManagerConfigurationType from ConfigInterface
```

Comment:
Works for me in `test` directory as we have service manager as `dev` dependency 👍🏼